api/grpcserver/globalstate_service.go

```diff
@@ -235,6 +236,9 @@ func (s GlobalStateService) AccountDataStream(in *pb.AccountDataStreamRequest, s
 			rewardsCh, rewardsBufFull = consumeEvents(stream.Context(), rewardsSubscription)
 		}
 	}
+	if err := stream.SendHeader(metadata.MD{}); err != nil {
```

Comment:
curious what difference does this make?

Reply:
it allows to wait for a confirmation that stream subscribed for events